README.md

```diff
@@ -114,14 +114,14 @@ io.opentracing.Tracer tracer = ...;
 // STEP 1 ABOVE: start the Scope/Span
 try (Scope scope = tracer.buildSpan("ServiceHandlerSpan").startActive(false)) {
     ...
+    // STEP 2 ABOVE: Implicitly pass the span to the next thread via shared value.
     final Span span = scope.span();
     doAsyncWork(new Runnable() {
         @Override
         public void run() {
-
-            // STEP 2 ABOVE: reactivate the Span in the callback, passing true to
-            // startActive() if/when the Span must be finished.
-            try (Scope scope = tracer.scopeManager().activate(span, false)) {
+            // STEP 3 ABOVE: reactivate the Span on the new thread, passing true to
+            // startActive(). Span will be closed when try block completes.
```

Comment:
And then "finish" is like "span is completed and cannot be reactivated" and "activate(span, true)" says "after scopeManager.close() is called here, call span.finish() and you can't use this span after this point"?

What's the difference between "span will be closed" and "span will be finished"?